src/styles/blocks/card.scss

```
@@ -0,0 +1,84 @@
+.card {
+  width: $cardWidth;
+  padding: 29px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 11px;
+
+  &__image {
+    margin-bottom: 45px;
+    width: 160px;
+    height: 134px;
+    transition-duration: 400ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__name {
+    margin-bottom: 6px;
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-bottom: 15px;
+    font-weight: 400;
+    align-self: flex-start;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    align-self: stretch;
+    margin-bottom: 20px;
+  }
+
+  &__comments {
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-self: stretch;
+    align-items: baseline;
+    padding-bottom: 20px;
+  }
+
+  &__word {
+    font-weight: 400;
+  }
+
+  &__amount {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__button {
+    text-align: center;
+    text-decoration: none;
+    background-color: $main-color;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    height: 40px;
+    line-height: 38px;
+    border-radius: 5px;
+    width: 170px;
+    border: 1px solid $main-color;
+    transition-duration: 400ms;
```

Always list styles that you apply transition for:
```suggestion
    transition: background-color 400ms, color 400ms;
```